tests/testutils.py

```diff
@@ -20,14 +20,36 @@
 
 
 class BaseTestCase(SimpleTestCase):
-    def tearDown(self) -> None:
+    def setUp(self):
+        super().setUp()
+        self._start_gen_id_patch()
+
+    def tearDown(self):
+        self._stop_gen_id_patch()
+
         super().tearDown()
         registry.clear()
 
         from django_components.template import _create_template
 
         _create_template.cache_remove()  # type: ignore[attr-defined]
 
+    # Mock the `gen_id` function so it returns deterministic IDs
+    def _start_gen_id_patch(self):
```

Comment:
Snce I'm using the UUIDs from nanoid, I needed to mock the `gen_id` function (which itself calls `nanoid`), so that the component IDs in the tests are deterministic.